### PR TITLE
behave test source command

### PR DESCRIPTION
**Pull request by meeuw**
## Description
behave test source command

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by j-bennet** [comment]
One of the tests actually breaks for me, I think we need to update expected help output [here](https://github.com/dbcli/pgcli/blob/6f1f4bfa8e96cfadded02ac8c7554f1c61c5446f/tests/features/fixture_data/help_commands.txt#L5):

```
Scenario: run source command  # features/basic_commands.feature:15
    When we run dbcli           # features/steps/basic_commands.py:16 0.003s
    And we wait for prompt      # features/steps/basic_commands.py:28 0.361s
    And we send source command  # features/steps/basic_commands.py:53 0.240s
    Then we see help output     # features/steps/crud_database.py:72 1.001s
      Traceback (most recent call last):
        File "/Users/irina/.pyenv/versions/2.7.12/envs/pgcli/lib/python2.7/site-packages/behave/model.py", line 1456, in run
          match.run(runner.context)
        File "/Users/irina/.pyenv/versions/2.7.12/envs/pgcli/lib/python2.7/site-packages/behave/model.py", line 1903, in run
          self.func(context, *args, **kwargs)
        File "features/steps/crud_database.py", line 75, in step_see_help
          wrappers.expect_exact(context, expected_line, timeout=1)
        File "/Users/irina/src/pgcli/tests/features/steps/wrappers.py", line 15, in expect_exact
          actual))
      Exception: Expected:
      ---
      u'\d [pattern]'
      ---

      Actual:
      ---
      u"             | Change to a new database.                     |
      | \copy [tablename] to/from [filename] | Copy data between a file and a table.         |
      | \d[+] [pattern]                      | List or describe tables, views and sequences. |
      | \dT[S+] [pattern]                    | List data types                               |
```

How did it not break in travis?

**Comment by meeuw** [comment]
Could you please have a look at ```/tmp/dmtest``` what's exactly outputted in the help output? I don't get this error either on my system.

File: tests/features/steps/basic_commands.py [hunk 1]
```diff
@@ -19,6 +20,7 @@ def step_run_cli(context):
     """
     cli_cmd = context.conf.get('cli_command')
     context.cli = pexpect.spawnu(cli_cmd, cwd='..')
+    context.cli.logfile = open('/tmp/dmtest', 'a')
```
Comment (meeuw) [inline tests/features/steps/basic_commands.py]: Reminder: remove this line

**Comment by j-bennet** [comment]
So I think my tests broke because of this:

https://github.com/dbcli/pgspecial/pull/31

Before testing this, I installed pgspecial into pgcli venf to test the pgspecial PR. It's not a problem.

**Comment by j-bennet** [comment]
All good now. Merging.

**Pull request merged**